conf/mraData/metaData.json

```diff
@@ -8,6 +8,6 @@
       "ja": "Machine Readable Appendix V1.1.1",
       "en": "Machine Readable Appendix V1.1.1"
     },
-    "Copyright": "(C) 2021-2022 Kanagawa Institute of Technology, ECHONET Consotrium"
+    "Copyright": "(C) 2021-2023 Kanagawa Institute of Technology, ECHONET Consotrium"
```

Comment:
@justdecodeme Please revert this back as it's an external data library, not our application codebase.

Reply:
Fixed. ✅